lib/tasks/deployment/20231125151610_set_case_contacts_as_active.rake

```diff
@@ -3,7 +3,11 @@ namespace :after_party do
   task set_case_contacts_as_active: :environment do
     puts "Running deploy task 'set_case_contacts_as_active'"
 
-    CaseContact.all.each do |cc|
+    CaseContact.where.not(casa_case_id: nil).each do |cc|
```

Comment:
In case users can already create legit drafts in prod. This would stop it for failing for drafts that would fail other validations.